lib/Conversion/VerifToSMT/VerifToSMT.cpp

```diff
@@ -208,6 +209,7 @@ struct VerifBoundedModelCheckingOpConversion
 
     unsigned numRegs =
         cast<IntegerAttr>(op->getAttr("num_regs")).getValue().getZExtValue();
+    auto initialValues = cast<ArrayAttr>(op->getAttr("initial_values"));
```

Comment:
There should be an ODS generated getter for the attribute. It's not great to hardcode the attribute name here.

Reply:
Good point, I'll swap that and the num_regs one over!